packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -221,6 +221,29 @@ abstract class MaterialLocalizations {
   /// Full unabbreviated year format, e.g. 2017 rather than 17.
   String formatYear(DateTime date);
 
+  /// Formats the date in a compact format.
+  ///
+  /// Usually just the numeric values for the for day, month and year are used.
+  ///
+  /// Examples:
+  ///
+  /// - US English: 02/21/2019
+  /// - Russian: 21.02.2019
+  ///
+  /// See also:
+  ///   * [parseCompactDate], which will convert a compact date string to a [DateTime].
+  String formatCompactDate(DateTime date);
+
+  /// Formats the date using a short-width format.
+  ///
+  /// Includes the abbreviation of the month, the day and year.
+  ///
+  /// Examples:
+  ///
+  /// - US English: Feb 21, 2019
+  /// - Russian: 21 февр. 2019 г.
```

Comment:
No, but I used the existing date formatters the the intl package to generate them, so I assumed it would be correct 😄. That said, I will ask around and see if anyone can review these.

Reply:
I'm a native Russian speaker and I approve this format :) (caveat: I'm not an expect on proper date formatting in Russia though)

Reply:
Maybe @mraleph can double-check.

Reply:
Looks fine to me, though a tad bizarre. I thought abbreviation for February uses 3 letters rather than 4, but searching around reveals that people use 4... Well, I learned something today... Maybe I was asleep on that lesson in school :)